apps/vercel/src/inngest/functions/users/sync-users-page.ts

```diff
@@ -0,0 +1,105 @@
+import { Elba, type User } from '@elba-security/sdk';
+import { NonRetriableError } from 'inngest';
+import { eq } from 'drizzle-orm';
+import { type VercelUser, getUsers} from '@/connectors/users'; 
+import { db } from '@/database/client';
+import { Organisation } from '@/database/schema';
+import { env } from '@/env';
+import { inngest } from '@/inngest/client';
+
+const formatElbaUser = (user: VercelUser): User => ({
+  id: user.uid,
+  displayName: user.name,
```

Comment:
```suggestion
  displayName: user.name || user.email,
```
Tried it and sometimes name is undefined